README.md

```diff
@@ -13,6 +13,13 @@ Both systems boot autonomously and can be controlled via an external debugger.
 Please read the section corresponding to the kit you are interested in for
 instructions on how to use this repo.
 
+After first cloning the freedom repository, you may need to run the following
```

Comment:
this is good, but why not just give all the commands for how to clone and use this repo? Since new users may not be familiar with git at all.

Reply:
Okay, I'll do that!